### ISSUE TITLE
DB/Creature: Strigid Screecher should get assists

### ISSUE THREAD

**Issue by Teppic1** [TrinityCore/TrinityCore#24257]
**Description:**

These owls (id 1996) will pull any others nearby during combat, on a repeat timer every few seconds. Tested in Classic retail, it's not just an aggro pull as it happens if you kite them in range of others too.

**Current behaviour:**

No script.

**Branch(es):**

3.3.5

**TC rev. hash/commit:** 

c5226aec41ce1430b53cc5b4073dff644fe37dac

Note: haven't tested Strigid Hunters if they have the same behaviour. (I don't know how factions work so I haven't attempted a fix)

**Comment by Aokromes** [comment]
a lot of start zone assists where removed on wow 3.

**Comment by Teppic1** [comment]
I've no way of checking if it was removed as I can't even find any comments about them doing it in the first place. These are level 7-8 though, not 1-5, if that makes any difference.

**Comment by illfated** [comment]
That is an interesting point. I can't remember seeing that behaviour myself, but then again I did not fight those birds much on retail back in 2006-2012. I wonder if a similar behaviour would apply to other beasts who use the emote "%s cries out for help."

**Comment by Teppic1** [comment]
In this case there's no message about assistance. I noticed it because all of the owls are neutral normally, so it's obvious when another you haven't hit suddenly starts attacking you.

**Comment by Killyana** [comment]
This is an old behavior that was removed after vanilla.

**Comment by Teppic1** [comment]
Source would be nice...

**Comment by Killyana** [comment]
I just tested in retail and it doesn't happen, blizzard made many changes in starting zones after vanilla and BC, but finding any sources would be hard.

**Comment by illfated** [comment]
It is even harder to find because the behaviour is for mobs above level 1-5 (after leaving the sandbox starter area). The changes for mobs in the sandbox starter area (1-5) is documented well enough where all mobs became neutral instead of hostile.

**Comment by Aokromes** [comment]
https://us.forums.blizzard.com/en/wow/t/starting-area-neutral-mobs/274573

**Comment by Teppic1** [comment]
Yeah, in 3.3.0 they made the 1-5 experience easier by making the hostile mobs now neutral in all of the starter areas.

I don't know of any changes to mobs outside of the starter areas though. I don't think any hostile mobs were made neutral and I don't know of any abilities being removed. In Cata tons of mobs were changed, including abilities/loot, so current retail isn't a good source for earlier.

**Comment by illfated** [comment]
Yes, that is the easy part. Mobs in the Starter area (Deathknell, Northshire Abbey, Valley of Trials, Coldridge Valley, Red Cloud Mesa, Shadowglen) became 100% neutral in 3.3.0, while before that they had some hostile mobs level 4.5.

This issue is not about starter areas named above, it is about the level 6+ mobs outside of those mini-zones (areas).

**Comment by Teppic1** [comment]
As for the owls, they don't spawn very close together and they don't emote anything so more often than not you won't see this behaviour or notice it when it happens. As these aren't unique/elites I can't see them being changed until Cata, though this is more of a general point rather than saying these are extremely important.

**Comment by illfated** [comment]
Fair enough. I see what you are saying. I just hope someone else will come across reliable information on this behaviour, because to me, it feels like it belongs in there somewhere (and not necessarily limited to the Strigid Screechers).

**Comment by Killyana** [comment]
This happen in cata, retail, and classic so probably also must happen in wotlk:
https://imgur.com/8SKgYmB

**Comment by Teppic1** [comment]
Additionally in Classic I noticed that neutral low level wolves in Elwynn Forest will assist higher level hostile wolves. Like other 1-4 level neutral mobs though they don't assist each other.

**Comment by illfated** [comment]
Isn't that implemented already in TC? (I could be mistaken, of course)

**Comment by Teppic1** [comment]
The wolves (mangy, young scavenger) don't have any SAI so it seems not. It may be related in that neutral mobs will assist the same faction but only if they're level 5+.

**Comment by illfated** [comment]
Good point. That is very likely the behaviour I have observed.

**Comment by dr-j** [comment]
I have seen neutral fearie dragons in ferelas cause nearby neutral trees to also agro and heal the fearie dragon

**Comment by illfated** [comment]
Nice and interesting. That would definitely add to the game world experience and immersement.

**Comment by offl** [comment]
So #17713 is invalid?

**Comment by illfated** [comment]
Hmm. I am uncertain, because I remember master branch having some form of aggro issue, where the neutral mobs (even level 1-3) would attack a player, just as if it was a real hostile mob. I have no idea what the status of the master branch is now, but I think it was a temporary issue several years ago.

**Comment by dr-j** [comment]
We do have a few periodic agro spells in db maybe one if these or similar

**Comment by offl** [comment]
> This happen in cata, retail, and classic so probably also must happen in wotlk:
> https://imgur.com/8SKgYmB

It's combat pulse

**Comment by offl** [comment]
https://youtu.be/90hcyilsu9Q?t=97
For some reason these won't aggro together. So it's probably indeed combat pulse since it cannot be called instantly.

CallForAssist is called instantly(its effect is delayed of course) and only once after entering combat and should be called even if creature was one-shoted. As you can see in video nothing like this happened so it's completely not related to CallForAssist.
CallForAssist is not called currently for neutral mobs and probably should never be called.

Seems like nobody yet opened issue about completely missing combat pulse for any kind of mobs because it's not related to faction at all, it's just general mechanic that works with ALL mobs.

I guess all we need here is timers for combat pulse and range of effect.

**Comment by illfated** [comment]
Good point. I sometimes see non-elite mobs anywhere between level 25 and 45 running to beg for help before they die, but even if they reach their nearest faction ally, it often does not seem like they manage to enlist the other one to attack my character. It is much more common for more mobs to aggro because of aggro chaining, as far as I have noticed.

**Comment by offl** [comment]
Flee is different thing, I will be not surprised if during flee combat pulse timer will not tick at all.
Btw even if fleeing creature reaches nearest ally there is a pause before engage

**Comment by offl** [comment]
Anyone can test this https://youtu.be/GzbUL5ytabI?t=102 on retail just by casting fear on any creature in dungeon? I have a feeling it's just combat pulse but I have to be sure. If feared creature always (and almost instantly) aggroes other mobs when nearby, it's not combat pulse.
